psi-ms.obo

```diff
@@ -22634,25 +22654,77 @@ relationship: has_metric_category MS:4000012 ! single run based metric
 relationship: has_units UO:0000189 ! count unit
 relationship: has_value_type xsd:integer ! The allowed value-type for this CV term
 
-[Term]
-id: MS:4000102
-name: number of detected quantification data points
-def: "The number of datapoints detected for quantification purposes within the run. These datapoints may be for example XIC profiles, isotopic pattern areas, or reporter ions (see MS:1001805). The used type should be noted in the meta data or analysis methods section of the recording file for the respective run." [PSI:MS]
-is_a: MS:4000003 ! single value
+[Term] 
+id: MS:4000102 
+name: number of detected quantification data points 
+def: "The number of datapoints detected for quantification purposes within the run. These datapoints may be for example XIC profiles, isotopic pattern areas, or reporter ions (see MS:1001805). The used type should be noted in the meta data or analysis methods section of the recording file for the respective run." [PSI:MS] 
```

Comment:
**nitpick:** "datapoints" → "data points", "meta data" → "metadata". Here and in the following terms.